tests/SmsLengthTest.php

```diff
@@ -84,7 +84,22 @@ public function providerSize()
             [str_repeat("exact•max", 67), 'ucs-2', 603, 9, 603],
 
             // empty
-            ['', '7-bit', 0, 1, 160]
+            ['', '7-bit', 0, 1, 160],
+
+            [
```

Comment:
Could you add a comment explaining what the test is checking, please? Something which explains the value, like in #1 would be great:

> The '[' at character 152 will become "0x1B[" (2 characters), and become part of the 2nd message, which pushes the last "." at the end of the content into a 3rd message